gui/src/components/Taipy/Image.tsx

```diff
@@ -147,18 +129,18 @@ const Image = (props: ImageProps) => {
     return (
         <Tooltip title={hover || ""}>
             <ImageButton
-                focusRipple
+                focusRipple={!!onAction}
                 style={style}
                 onClick={handleClick}
-                disabled={!active}
+                disabled={!active || !onAction}
                 className={className}
                 id={id}
             >
                 <ImageSrc style={imgStyle} />
-                <ImageBackdrop className="MuiImageBackdrop-root" />
+                {onAction ? <ImageBackdrop className="MuiImageBackdrop-root" /> : null}
                 {label === undefined ? null : (
                     <ImageSpan>
-                        <Typography component="span" variant="subtitle1" color="inherit" sx={imgSx}>
+                        <Typography component="span" variant="subtitle1" color="inherit" sx={imgSx} className={getSuffixedClassNames(className, "-label")}>
```

Comment:
This -label thing should be documented somehere right?
Can you tell me about it?
Thx!

Reply:
for ease of use, we append "-label" to each css class declared for the component (taipy-image + the user specified ones)

Reply:
I guess it should be more widespread ...
It has been done in a few other places.